### PR TITLE
[bulk] validation: fix duplicateH1Title issues and description-missing issues.

### PR DESCRIPTION
**Pull request by Vette99**
This PR fixed 40 duplicateH1Title issues and 20 description-missing issues and 5 html issues.

### CODE REVIEW

File: Skype/SDN/articles/getting-ready-to-install-skype-for-business-sdn-interface.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ﻿---
-title: Getting ready to install Skype for Business SDN Interface
+title: Getting ready for installing Skype for Business SDN Interface
```
Comment (dereklh7) [inline Skype/SDN/articles/getting-ready-to-install-skype-for-business-sdn-interface.md:2]: Why change the title? This makes the title inconsistent with the H1.

File: Skype/SDN/articles/getting-ready-to-install-sdn-interface.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Getting ready to install Skype for Business SDN Interface
- 
+title: Getting ready to install Skype for Business SDN Interface
```
Comment (dereklh7) [inline Skype/SDN/articles/getting-ready-to-install-sdn-interface.md:2]: Getting ready to install Skype for Business SDN - make this the title and H1.

File: Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-d.md [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
 ms:contentKeyID: 65855446
+description: The source of the media stream.
 ms.date: 08/24/2015
 mtps_version: v=office.16
 dev_langs:
 - xml
 ---
 
-# From element 
+# From element ((Schema D))
```
Comment (dereklh7) [inline Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-d.md:14]: Make the H1 consistent with the title.

File: Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md [hunk 4]
```diff
@@ -1,16 +1,17 @@
 ﻿---
-title: From element (StartOrUpdateType complexType) 
+title: From element (StartOrUpdateType complexType) (Schema C)
```
Comment (dereklh7) [inline Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md:2]: Make title and H1 consistent.

File: Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md [hunk 5]
```diff
@@ -1,16 +1,17 @@
...
 TOCTitle: From element (StartOrUpdateType complexType)
 ms:assetid: 75f762ae-fe25-fc78-7b61-57d4a6464f20
 ms:mtpsurl: https://msdn.microsoft.com/library/Mt404765(v=office.16)
 ms:contentKeyID: 68250711
+description: Source of the media stream.
```
Comment (dereklh7) [inline Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md:7]: Descriptions should be 100-150 characters. Too short as is.

File: Skype/SDN/articles/hostip-element-byeendpointtype-complextype-skype-for-business-sdn-interface-2-2-schema-d.md [hunk 6]
```diff
@@ -1,16 +1,17 @@
 ﻿---
-title: HostIP element (ByeEndPointType complexType) 
+title: HostIP element (ByeEndPointType complexType) (Schema D)
 TOCTitle: HostIP element (ByeEndPointType complexType)
 ms:assetid: 6e4a481a-9960-5cf1-afdb-00bbba051c02
 ms:mtpsurl: https://msdn.microsoft.com/library/Mt170880(v=office.16)
 ms:contentKeyID: 65855454
+description: IP address of the endpoint's host in the local network.
```
Comment (dereklh7) [inline Skype/SDN/articles/hostip-element-byeendpointtype-complextype-skype-for-business-sdn-interface-2-2-schema-d.md:7]: Description is too short.

**Comment by Vette99** [comment]
@lindalu-MSFT
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

File: Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md (hunk 5 above)
Comment (Vette99) [reply Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md:7]: Not enough info in article to create a 100 character description.

File: Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md (hunk 4 above)
Comment (Vette99) [reply Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md:2]: H1s and Title consistentency has not been flagged as an error or suggestion or warning.

**Pull request merged**